ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java

```diff
@@ -1,29 +1,30 @@
 package org.openmrs.reference;
 
+import java.util.List;
+
 import org.junit.Assert;
-import org.junit.Ignore;
 import org.junit.experimental.categories.Category;
+import org.junit.jupiter.api.Test;
+import org.openmrs.reference.groups.BuildTests;
 import org.openmrs.reference.page.AdministrationPage;
 import org.openmrs.reference.page.ModulesPage;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 
-import java.util.List;
-
 public class CheckModulesTest extends ReferenceApplicationTestBase {
 
     /**
      * Check the list of modules to be sure they are all started.
      */
-    @Ignore
-    @Category(org.openmrs.reference.groups.BuildTests.class)
-    public void checkModules() throws Exception {
+    @Test
+    @Category(BuildTests.class)
+    public void checkModulesTest() {
```

Comment:
Is there any reason for changing the original method name?

Reply:
Selenium legacy ui tests method naming follow a convention of naming the test method the `testClassName` except beginning with lower Case letter as embraced by method naming  in java programming !

That side, already existing test method names gives a hint why changing the original method name to checkModuleTest() was done. 
Below is are four samples of existing method naming !
[addUserTest()](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/a2808ee171e7dca2c8bc82acef722971f233c32e/ui-tests/src/test/java/org/openmrs/reference/AddUserTest.java#L31) for AddUserTest class
[endVisitTest()](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/a2808ee171e7dca2c8bc82acef722971f233c32e/ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java#L36) for EndVisitTest class
[searcActiveVisitTest()](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/a2808ee171e7dca2c8bc82acef722971f233c32e/ui-tests/src/test/java/org/openmrs/reference/SearchActiveVisitsTest.java#L36) for SearchActiveTest class

[xssOnPhoneNumberFieldTest()](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/a2808ee171e7dca2c8bc82acef722971f233c32e/ui-tests/src/test/java/org/openmrs/reference/XSSOnPhoneNumberFieldTest.java#L36)